l10n_ch_fds_upload_sepa/wizard/fds_inherit_sepa_wizard.py

```diff
@@ -0,0 +1,319 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Swiss Postfinance File Delivery Services module for Odoo
+#    Copyright (C) 2014 Compassion CH
+#    @author: Nicolas Tran
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api, exceptions
+import logging
+import base64
+import tempfile
+import shutil
+import pysftp
+import os
+
+_logger = logging.getLogger(__name__)
+
+
+class fds_inherit_sepa_wizard(models.TransientModel):
+    ''' This addons allows you to upload the generate SEPA electronic payment
+        file for Switzerland to your FDS Postfinance.
+
+        This addon will add upload button to the SEPA wizard.
+        This Class inherit from l10n_ch_sepa.wiz_pain_001.
+    '''
+    _inherit = 'wizard.pain001'
+
+    fds_account = fields.Many2one(
+        comodel_name='fds.postfinance.account',
+        string='FDS account',
+        help='select one FDS account or create a FDS account'
+    )
+    fds_account_directory = fields.Many2one(
+        comodel_name='fds.postfinance.files.directory',
+        string='FDS files directory',
+        help='select one upload directory or config your upload directory'
+    )
+    attachment_id = fields.Many2one(
+        comodel_name='ir.attachment',
+        string='File attachment',
+        readonly=True,
+        help='[info] keep one recored of the model attachment'
+    )
+    filename = fields.Char(
+        string='Attachment filename',
+        related='attachment_id.name',
+        readonly=True,
+        help='The name of the attachment file'
+    )
+    payment_order_id = fields.Many2one(
+        comodel_name='payment.order',
+        string='Payment order',
+        readonly=True,
+        help='payment order id'
+    )
+    state = fields.Selection(
+        selection=[('default', 'Default'),
+                   ('finish', 'finish'),

```

Comment:
It would be better with capital letter at the beginning of displayed word to the user.